CHANGES.md

```diff
@@ -1,3 +1,12 @@
+## v5.0.0
+Since we have sunsetted Stories, we have removed `converse` and `runActions`. You can learn more about how to migrate your apps here: https://github.com/wit-ai/wit-stories-migration-tutorial
```

Comment:
The most important change is the removal of `.converse()` and `runActions()`. Follow the migration tutorial [here](https://github.com/wit-ai/wit-stories-migration-tutorial), or [read more here](https://wit.ai/blog/2017/07/27/sunsetting-stories).